src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -414,6 +416,12 @@ blockCommands = M.fromList $
   , "pagebreak"
   ]
 
+blockTextcolor :: PandocMonad m => LP m Blocks
+blockTextcolor = do
+    skipopts 
+    color <- braced
+    divWith ("",[],[("style","color: " ++ color)]) <$> grouped block <* notFollowedBy inline
+
```

Comment:
I assume that you think about following case, right?

```latex
\textcolor{red}{
\begin{itemize}
    \item Test 1
    \item Test 2
\end{itemize}
} some more text
```
I tried to add `try` before, however, this fix is not working (see diff). 

```diff
diff --git a/src/Text/Pandoc/Readers/LaTeX.hs b/src/Text/Pandoc/Readers/LaTeX.hs
index a34be46e..85a77354 100644
--- a/src/Text/Pandoc/Readers/LaTeX.hs
+++ b/src/Text/Pandoc/Readers/LaTeX.hs
@@ -417,7 +417,7 @@ blockCommands = M.fromList $
   ]
 
 blockTextcolor :: PandocMonad m => LP m Blocks
-blockTextcolor = do
+blockTextcolor = try $ do
     skipopts 
     color <- braced
     divWith ("",[],[("style","color: " ++ color)]) <$> grouped block <* notFollowedBy inline
```

I think, that using `notFollowedBy inline` need to be changed. LaTeX puts _some more text_ into a new paragraph. Pandoc should do the same thing. Do you agree?